aws/resource_aws_lb_listener_rule_test.go

```diff
@@ -2099,3 +2526,292 @@ resource "aws_security_group" "test" {
 }
 `, rName)
 }
+
+func testAccAWSLBListenerRuleConfig_conditionAttributesCount_pairs() []string {
+	inputs := map[string]string{
+		"host_header": `= ["example.com"]`,
+		"http_header": `{
+		  http_header_name = "X-Clacks-Overhead"
+		  values = ["GNU Terry Pratchett"]
+		}`,
+		"http_request_method": `= ["POST"]`,
+		"path_pattern":        `= ["/"]`,
+		"query_string": `{
+		  key = "foo"
+		  value = "bar"
+		}`,
+		"source_ip": `= ["192.168.0.0/16"]`,
+		"field": `= "host-header"
+		  values = ["example.com"]`,
+	}
+
+	// Extract keys
```

Comment:
These for loops generate 21 conditions to test. Expanding those out to full tests will be both boring and error prone. Should a new condition need to be added it will also be error prone.

I'd rather just delete the test. It's only guarding against future changes to the code base, specifically `lbListenerRuleConditions` method and not compatibility with AWS's API.

Reply:
Or have a single test for each condition type against just one other instead of an exhaustive suite. Currently that would only need 6 tests.

Reply:
That last sounds like a reasonable option to me, thanks!

Reply:
OK. I've got rid of this test generation.